integration-tests/tests/preview-dataset.js

```diff
@@ -7,7 +7,7 @@ const datasetName = 'Readable Catalog Snapshot 1'
 
 const testPreviewDatasetFn = withUserToken(async ({ testUrl, page, token }) => {
   await enableDataCatalog(page, testUrl, token)
-  await click(page, clickable({ textContains: 'browse & explore' }))
+  await click(page, clickable({ textContains: 'datasets' }))
```

Comment:
Had to make this change because the new browser page is intended to replace the datasets page and the mockups have now removed the "browse and explore" tab